engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql

```diff
@@ -158,6 +158,298 @@ WHERE
     name IN ("quota.usage.smtp.useStartTLS", "quota.usage.smtp.useAuth", "alert.smtp.useAuth", "project.smtp.useAuth")
     AND value NOT IN ("true", "y", "t", "1", "on", "yes");
 
+CREATE TABLE `cloud`.`shared_filesystem`(
+    `id` bigint unsigned NOT NULL auto_increment COMMENT 'ID',
+    `uuid` varchar(40) COMMENT 'UUID',
+    `name` varchar(255) NOT NULL COMMENT 'Name of the shared filesystem',
+    `description` varchar(1024) COMMENT 'Description',
+    `domain_id` bigint unsigned NOT NULL COMMENT 'Domain ID',
+    `account_id` bigint unsigned NOT NULL COMMENT 'Account ID',
+    `data_center_id` bigint unsigned NOT NULL COMMENT 'Data center ID',
+    `state` varchar(12) NOT NULL COMMENT 'State of the shared filesystem in the FSM',
+    `fs_provider_name` varchar(255) COMMENT 'Name of the shared filesystem provider',
+    `protocol` varchar(10) COMMENT 'Protocol supported by the shared filesystem',
+    `volume_id` bigint unsigned COMMENT 'Volume which the shared filesystem is using as storage',
+    `vm_id` bigint unsigned COMMENT 'vm on which the shared filesystem is hosted',
+    `fs_type` varchar(10) NOT NULL COMMENT 'The filesystem format to be used for the shared filesystem',
+    `service_offering_id` bigint unsigned COMMENT 'Service offering for the vm',
+    `update_count` bigint unsigned COMMENT 'Update count for state change',
+    `updated` datetime COMMENT 'date updated',
+    `created` datetime NOT NULL COMMENT 'date created',
+    `removed` datetime COMMENT 'date removed if not null',
+    PRIMARY KEY (`id`),
+    CONSTRAINT `uc_shared_filesystem__uuid` UNIQUE (`uuid`),
+    INDEX `i_shared_filesystem__account_id`(`account_id`),
+    INDEX `i_shared_filesystem__domain_id`(`domain_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP VIEW IF EXISTS `cloud`.`shared_filesystem_view`;
```

Comment:
these views should be removed from schema-41910to42000.sql

Reply:
Done